pandas/core/internals/managers.py

```diff
@@ -1080,7 +1082,7 @@ def iset(
             blkno = self.blknos[loc]
             blk = self.blocks[blkno]
             if len(blk._mgr_locs) == 1:  # TODO: fastest way to check this?
-                return self._iset_single(
+                self._iset_single(
```

Comment:
Do we still want a return here?

Reply:
Yes, you are right! I thought I can remove it because it returns "None" but it also ends the execution of this method!